dev/run-python-skinny-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Ditto here - can we add a small README at the top of this file explaining the purpose of skinny client testing?